functions/use_firewalld.pp

```diff
@@ -0,0 +1,41 @@
+# Returns ``true`` if the client can/should use firewalld
+#
+# @param enable
+#   The type of enablement to use
+#
+#   * true      => Do the right thing based on the underlying OS
+#   * false     => Return `false`
+#   * firewalld => Force `firewalld` if available
+#
+# @return [Boolean]
+#
+function iptables::use_firewalld(
```

Comment:
Are you assuming this is fully tested in the acceptance test?  Didn't see a unit test.

Reply:
The unit tests are through the Hieradata in the 'listen/firewalld' defined type tests.